### ISSUE TITLE
Взаимодействие игроков: гильдии и альянсы

### ISSUE THREAD

**Issue by m1kc** [uonline/universe#47]
В первом приближении это выглядит как-то так.

Люди объединяются в гильдию, чтобы заниматься каким-то общим делом (например, в гильдии алхимиков могут могут быть люди, занимающиеся сбором редкой травы по всему игровому миру и люди, у которых вкачан навык варить из неё что-то годное). За создание гильдии взимается некоторое количество золота. У гильдии есть банк и внутренняя почта. Возможно, одному человеку можно позволить состоять в нескольких гильдиях.

Люди объединяются в альянс, чтобы воевать. У альянса есть иерархия, звания, должности. За создание альянса взимается куда больший взнос, и необходимы подписи сразу, скажем, 20 человек. У альянса есть внутренняя почта и банк - меньшего размера, чем у гильдии. Человек может состоять только в одном альянсе.

**Comment by Nowayinrf** [comment]
Так, в итоге, какая основная разница между гильдией и альянсом? Гильдия - сбор по одной профе ?

**Comment by m1kc** [comment]
А тут обсуждение как-то встало. Вопрос не горит, успеем ещё решить.

**Comment by FaNatizM** [comment]
> гильдия — это не клуб по интересам. мол, "о, вы воры! и я — вор, ай да к вам". ему бы сказали: "нет, изволь доказать". и тогда надо зарабатывать репутацию вора, получать задания, за провал задания — штраф очков репутации. и потом — получить какую-то уникальную для гильдии вещь. и обязательно с прямым запретом на отчуждение (невозможно продать, только выкинуть, при этом вещица перестает существовать в мире, при поражении — не может перейти к сопернику).
